db/structure.sql

```diff
@@ -0,0 +1,148 @@
+--
+-- PostgreSQL database dump
```

Comment:
👍  for sql structure. We should probably remove the schema.rb in this PR though.